src/test/java/org/sagebionetworks/bridge/services/StudyServiceTest.java

```diff
@@ -771,7 +779,14 @@ public void transitionToRecruitment() {
         study.setScheduleGuid(SCHEDULE_GUID);
         when(mockStudyDao.getStudy(TEST_APP_ID, TEST_STUDY_ID)).thenReturn(study);
 
+        Assessment assessment = new Assessment();
+        assessment.setGuid(ASSESSMENT_1_GUID);
+        assessment.setPhase(AssessmentPhase.REVIEW);
```

Comment:
Is there a test somewhere in this file that checks that the assessment phase has actually changed to PUBLISHED after the study has moved to recruitment phase? Or maybe somewhere else? I don't remember seeing that anywhere but I might have just missed it.

Reply:
Just added that check to this test.